src/bot.js

```diff
@@ -3,138 +3,132 @@ const TelegramBot = require("node-telegram-bot-api");
   const chatID = msg.chat.id;
   const sessions = await db.readProgressFromFile();
 
   if (chatID in sessions) {
     bot.sendMessage(
       chatID,
-      "Давай продолжим. \nВот города которые были названы:\n" +
+      i18nt.t(lang, "CONT_LETS_PLAY") +
         [...sessions[chatID].spentCities].join(", ") +
-        "\nНужно назвать город на букву " +
+        i18nt.t(lang, "CONT_CITY_ON_LETTER") +
         sessions[chatID].lastLetter.toUpperCase()
     );
   } else {
-    bot.sendMessage(
-      chatID,
-      "Список доступных команд:\n/start\nНачать\nДля начала игры напиши Начать"
-    );
+    bot.sendMessage(chatID, i18nt.t(lang, "LIST_OF_COMMANDS"));
   }
 }
 
-bot.onText(commands.START_NEW_GAME, onNewGame);
 async function onNewGame(msg) {
+  const lang = msg.from.language_code;
   const chatID = msg.chat.id;
   let sessions = await db.readProgressFromFile();
 
   if (chatID in sessions) {
     await db.deleteSession(chatID, sessions);
     sessions = await db.makeSession(sessions, chatID);
-    startGame(chatID, sessions);
+    startGame(chatID, sessions, lang);
   } else {
-    bot.sendMessage(
-      chatID,
-      "Список доступных команд:\n/start\nНачать\nДля начала игры напиши Начать"
-    );
+    bot.sendMessage(chatID, i18nt.t(lang, "LIST_OF_COMMANDS"));
   }
 }
 
-bot.onText(commands.STOP_GAME, onStopGame);
 async function onStopGame(msg) {
+  const lang = msg.from.language_code;
   const chatID = msg.chat.id;
   const sessions = await db.readProgressFromFile();
 
   if (chatID in sessions) {
     bot.sendMessage(
       chatID,
-      "Я выиграл!!! \nГорода которые были названы:\n" + [...sessions[chatID].spentCities].join(", ")
+      i18nt.t(lang, "WIN_MESSAGE") + [...sessions[chatID].spentCities].join(", ")
     );
-    bot.sendMessage(chatID, "Давай еще сыграем! \nДля начала напиши Начать.");
+    bot.sendMessage(chatID, i18nt.t(lang, "LETS_PLAY_AGAIN"));
     await db.deleteSession(chatID, sessions);
   } else {
-    bot.sendMessage(
-      chatID,
-      "Список доступных команд:\n/start\nНачать\nДля начала игры напиши Начать"
-    );
-  }
-}
-
-bot.on("message", onMessage);
-async function onMessage(msg) {
-  const chatID = msg.chat.id;
-  for (const key in commands) {
-    if (msg.text.match(commands[key])) return;
-  }
-  const sessions = await db.readProgressFromFile();
-
-  if (chatID in sessions) {
-    processMessages(chatID, sessions, msg.text.toLowerCase());
-  } else {
-    bot.sendMessage(
-      chatID,
-      "Список доступных команд:\n/start\nНачать\nДля начала игры напиши Начать"
-    );
+    bot.sendMessage(chatID, i18nt.t(lang, "LIST_OF_COMMANDS"));
   }
 }
 
-async function startGame(chatID, sessions) {
-  const gStart = await game.start(chatID, sessions);
+async function startGame(chatID, sessions, lang) {
+  const gStart = await game.start(chatID, sessions, lang);
 
-  if (gStart.messages[1] === "Ошибка выбора города!") {
+  if (gStart.messages[1] === i18nt.t(lang, "ERR_SELECT_CITY")) {
```

Comment:
You could make better design. For example create botContext parameter that will be accepted. Pass there already localized function. Like

translate: (key) => i18n.t(lang, key)